src/TodoListItem.module.css

```diff
@@ -0,0 +1,51 @@
+.ListItem {
+  align-items: center;
+  color: #1a1a1a;
+  padding: 0.5rem;
+  list-style: none;
+  background-color: #84d8eb;
+  border: 5px solid #84d8eb;
+  margin-top: 1em;
+  border-radius: 4em;
+  box-shadow: 7px 7px 0px 0px rgba(54, 87, 235, 1);
+  font-weight: bold;
+  font-size: 18px;
+}
+
+/* CSS */
+.button54 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 10px;
+  letter-spacing: 2px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  border: 3px solid;
+  padding: 0.25em 0.5em;
+  box-shadow: 1px 1px 0px 0px, 2px 2px 0px 0px, 3px 3px 0px 0px,
+    4px 4px 0px 0px, 5px 5px 0px 0px;
+  position: relative;
+  user-select: none;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+  bottom: 3px;
+}
+
+.button54:active {
+  box-shadow: 0px 0px 0px 0px;
+  top: 5px;
+  left: 5px;
+}
+
+@media (min-width: 768px) {
+  .button54 {
+    padding: 0.25em 0.75em;
+  }
+}
```

Comment:
Looks like you're just getting started here with media queries. I encourage you to create more queries that actually accommodate smaller client sizes like tablet and mobile device widths.